chart/compass/values.yaml

```diff
@@ -150,6 +150,13 @@ global:
       name: compass-istio-gateway
       namespace: compass-system
 
+  gardener:
+    project: "kyma-dev" # Gardener project connected to SA
+    kubeconfigPath: "/gardener/kubeconfig/kubeconfig"
```

Comment:
this should not be a parameter, let's hardcode it in the chart (it is connected with volume mount)

Reply:
I have changed it to be built from configuration variable  .Values.global.gardener.kubeconfigMountPath